sdk/analytics/src/main/java/com/klaviyo/analytics/networking/KlaviyoApiClient.kt

```diff
@@ -117,6 +118,10 @@ internal object KlaviyoApiClient : ApiClient {
         apiObservers -= observer
     }
 
+    override fun getActiveForms() {
+        enqueueRequest(FormApiRequest())
+    }
+
```

Comment:
(question, non-blocking): I see we log something out before enqueuing the request in some of the other methods in here. Do we want to do that for `getActiveForms` too?

Reply:
Definitely not a bad idea, I can add that here